src/v0/sources/shopify/config.js

```diff
@@ -102,7 +102,11 @@ const SUPPORTED_TRACK_EVENTS = [
   'orders_paid',
   'orders_partially_fullfilled',
 ];
-
+const timeDifferenceForCartEvents = 10000; // in micro seconds as we will be compairing it in timestamp 
+const useRedisDatabase = process.env.USE_REDIS_DB === 'true' || true;
```

Comment:
By default have it as false as we are planning to test it in pro first & then enterprise
```suggestion
const useRedisDatabase = process.env.USE_REDIS_DB === 'true' || false;
```

Reply:
Done